### PR TITLE
Rucurring payments + payment data fixes

### PR DESCRIPTION
**Pull request by MartinMystikJonas**

### CODE REVIEW

File: src/Entity/Codes/PaymentMethodCode.php [hunk 1]
```diff
@@ -50,4 +50,5 @@ final class PaymentMethodCode
 
 	public const PART_TWISTO = 'PART_TWISTO';
 	public const PART_SKIPPAY = 'PART_SKIPPAY';
+
```
Comment (MartinMystikJonas) [inline src/Entity/Codes/PaymentMethodCode.php:53]: Coding standard fix

**Comment by f3l1x** [comment]
Do you plan to add some tests?

**Comment by MartinMystikJonas** [comment]
I can

**Comment by MartinMystikJonas** [comment]
Added tests

**Comment by f3l1x** [comment]
Awesome 👍

**Pull request merged**